cardano_node_tests/tests/test_kes.py

```diff
@@ -351,11 +368,17 @@ def test_opcert_future_kes_period(  # noqa: C901
                         shutil.copy(overincrement_opcert_file, opcert_file)
                         cluster_nodes.restart_nodes([node_name])
 
-                    if invalid_opcert_epoch == 3:
+                    if invalid_opcert_epoch == 3 and VERSIONS.cluster_era > VERSIONS.ALONZO:
                         # check kes-period-info with operational certificate with
-                        # invalid counter
-                        # TODO: the query is currently broken, implement once it is fixed
-                        pass
+                        # invalid counters
+                        kes_period_info = cluster.get_kes_period_info(overincrement_opcert_file)
```

Comment:
This check should be in the branch above (`if invalid_opcert_epoch == 2 and VERSIONS.cluster_era > VERSIONS.ALONZO:`) and here should be another check, both for Alonzo and Babbage eras.